.github/components.yml

```diff
@@ -0,0 +1,53 @@
+CPU:
+  dependent_components:
+    Python_API: [build, test]
+    C_API: [build, test]
+
+Python_API:
+  dependent_components:
+    CPU: [build]
+
+C_API:
+  dependent_components:
+    CPU: [build]
+
+Model_Hub_Tests:
+  dependent_components:
+    Torch_Model_Hub_Tests: [build, test]
+    TF_Model_Hub_Tests: [build, test]
+
+TF_FE:
+  dependent_components:
+    TF_Model_Hub_Tests: [build, test]
+
+PyTorch_FE:
+  dependent_components:
+    Torch_Model_Hub_Tests: [build, test]
+
+TFL_FE:
```

Comment:
It will affect all other components if it's not listed in a config. If it is listed, but without dependent_components - we consider that's intentional and validate changed component only.

I can change the behavior like that: if dependent_components key is not specified at all - consider that it affects all other components. If it **is** specified, but the value for this is empty - then it's most definitely intentional and means that it doesn't affect anything else

Reply:
Applied the change described above